resources/constants.py

```diff
@@ -856,6 +862,11 @@ def lang_enum(name: str, values: Sequence[str]) -> Dict[str, str]:
     'tfc.enum.nutrient.vegetables': 'Vegetables',
     'tfc.enum.nutrient.protein': 'Protein',
     'tfc.enum.nutrient.dairy': 'Dairy',
+    'tfc.enum.forgingbonus.none': 'No Forging Bonus',
+    'tfc.enum.forgingbonus.poorly_forged': 'Poorly Forged',
+    'tfc.enum.forgingbonus.well_forged': 'Well Forged',
+    'tfc.enum.forgingbonus.expertly_forged': 'Expertly Forged',
+    'tfc.enum.forgingbonus.perfectly_forged': 'Perfectly Forged!',
```

Comment:
> Show nothing if no bonus

It does, the enum translation key is included for completeness, since `Helpers.translateEnum(ForgingBonus)` could technically still be called elsewhere.